modules/dqt/js/arrayintersect.js

```diff
@@ -102,12 +102,15 @@ export const getSessions = function(group) {
 };
 
 export const enumToArray = function(enumString) {
-    let tempArray = enumString.split('(')[1].split('\'');
-    let array = [];
-    for (let i = 1; i < tempArray.length; i += 2) {
-        array.push(tempArray[i]);
-    }
-    return array;
+  enumString = enumString.substring('enum('.length); // remove 'enum('
+  enumString = enumString.slice(0, -1); // remove last ')'
+  const tempArray = enumString.split(/,\s*/);
```

Comment:
@maltheism thats not the regex I gave you...

This regex:
1. will erroneously match a coma inside one of the values. example `'rida, alizee and dave', 'samir', 'leigh'`
2. will not properly match and remove spaces before comas example `'rida' , 'alizee' , 'dave'`

Reply:
@ridz1208 the regex you gave me didn't work. I'll take a look at what I can do again.